src/ucp/proto/proto_am.inl

```diff
@@ -391,6 +391,8 @@ ucs_status_t ucp_do_am_zcopy_multi(uct_pending_req_t *self, uint8_t am_id_first,
                 }
                 ucs_assert(status == UCS_INPROGRESS);
                 return UCS_OK;
+            } else {
```

Comment:
no need for `else`
i.e. can fix it in the same way as in master:
https://github.com/openucx/ucx/blob/8d3801b94512e91c508a7d2a357b58bb9e5d4b6c/src/ucp/proto/proto_am.inl#L444